MAUI/PDF-Viewer/Printing-PDF-files.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Printing PDF files in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about printing feature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Printing PDF Files in .NET MAUI PDF Viewer (SfPdfViewer)
+
+The print feature of [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to effortlessly print PDF documents directly from your application code. Also, the modified form fields and annotations in SfPdfViewer will be preserved properly in the printed document.
+
+![Printing PDF Files in .NET MAUI PDF Viewer](Images\Print\print.jpg)
+
+Moreover, when performing a print operation, the default device print dialog opens, providing users with familiar options and settings to customize their printing experience. Within this dialog, users can adjust settings such as printer selection, the number of copies, page orientation, and more, ensuring that their printed documents meet their specific requirements.
+
+To print a PDF programmatically, you can use the `PrintDocument` method provided by SfPdfViewer. This method enables you to trigger the printing action seamlessly within your application.
```

Comment:
<img width="332" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/bcb1f86e-50a1-44cb-b150-047c5a78ef37">

Please avoid these terms when mentioning code and improve it.